src/platforms/hosted/dap_command.c

```diff
@@ -48,6 +48,8 @@
 #define DAP_JTAG_TMS_CLEAR   (0U << 6U)
 #define DAP_JTAG_TDO_CAPTURE (1U << 7U)
 
+#define DAP_TRANSFER_STATUS_MASK (0x7U)
```

Comment:
Please drop the parens - they're not required.

Reply:
done